src/Compilers/CSharp/Test/Emit3/Semantics/OutVarTests.cs

```diff
@@ -21562,96 +21565,128 @@ static bool TakeOutParam(object y, out bool x)
 }
 ";
             var compilation = CreateCompilationWithMscorlib461(source, options: TestOptions.DebugExe, parseOptions: TestOptions.Regular);
-            int[] exclude = new int[] { (int)ErrorCode.ERR_BadVarDecl,
```

Comment:
this test is insane.  it's testing out-vars in a series of invalidly written for-loops.  specifically, they are written as:

```c#
for (bool a, b(
             Dummy(TakeOutParam(1, out var x14), 
```

In the past, this was parsed terribly to begin with (since you had the varaibles `bool a, b` and then an error parentheszied expression following.

Now, this is at least seen as `bool a` as the variable, and `b(...)` as the condition of the for loop.  This ends up changing all  the binding.